eventmesh-connectors/eventmesh-connector-spring/src/test/java/org/apache/eventmesh/connector/spring/sink/connector/SpringSinkConnectorTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.connector.spring.sink.connector;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import org.apache.eventmesh.connector.spring.sink.config.SpringSinkConfig;
+import org.apache.eventmesh.openconnect.offsetmgmt.api.data.ConnectRecord;
+import org.apache.eventmesh.openconnect.offsetmgmt.api.data.RecordOffset;
+import org.apache.eventmesh.openconnect.offsetmgmt.api.data.RecordPartition;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.BlockingQueue;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class SpringSinkConnectorTest {
+
+    @Spy
+    private SpringSinkConnector connector;
+
+    @BeforeEach
+    public void setUp() throws Exception {
+        SpringSinkConfig sinkConfig = new SpringSinkConfig();
+        connector.init(sinkConfig);
+        connector.start();
+    }
+
+    @Test
+    public void springSinkConnectorTest() throws Exception {
+        final int count = 5;
+        final String message = "testMessage";
+        writeMockedRecords(count, message);
+        BlockingQueue<ConnectRecord> queue = connector.getQueue();
+        assertThat(count, is(queue.size()));
+        for (int i = 0; i < count; i++) {
+            ConnectRecord poll = queue.poll();
+            assertNotNull(poll);
+            String expectedMessage = message + i;
+            assertThat(poll.getData(), is(expectedMessage));
```

Comment:
Hamcrest's assertion is more powerful than junit5, and I think can be combined.

---

hamcrest的断言比junit5更强大, 我认为两者可以组合使用。

Reply:
Hamcrest is an additional dependency, and if we're using it solely for simple assertions, I believe that standard JUnit assertions should suffice. This can help us keep our project dependencies minimal and reduce complexity in our tests. This is just my opinion.

Reply:
I also think it's right to keep minimal dependencies, it has been fixed in new commit.